@hesburgh-wse/gatsby-theme-marble/src/components/Internal/CalloutBox/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import style from './style.module.css'
+
+const CalloutBox = ({ children }) => {
+  return (
+    <div
+      className={style.wrapper}
+      style={{
```

Comment:
Why the inline styles when this element has its own class?

Reply:
Because I forgot to commit the last change to this file, but it still has an `sx` prop and a `className` because some of this will be coming from `theme-ui`, but the `@media` styling is a little messy right now and it's easier to do it in css.